### ISSUE TITLE
Network service socket core dump on BMC image upload test via REST and TFTP server - OP910 1742E

### ISSUE THREAD

**Issue by cdeadmin** [openbmc/openbmc#2632]

**Comment by rfrandse** [comment]
<https://gerrit.openbmc-project.xyz/7863> Create netlink socket outside from rtnetlink server
Resolves  #2632 Network service socket core dump on BMC image upload test via REST and TFTP server - OP910 1742E

**Comment by rfrandse** [comment]
@ratagupt 
The network service socket core has not been fixed. Ratan is working on the fix and he will update once the fixes are ready.
~~~~
Create netlink socket outside from rtnetlink server As socket lifetime goes out of scope before starting the sd event loop,so now create the socket outside and pass it to rtnetlink server
